### PR TITLE
Iterating/Casting

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#409 
#### What does this implement/fix? Explain your changes.
- Custom iterating setting added
- Custom casting setting added

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@alirezazolanvari @sadrasabouri 
Please review this PR 🥇 
If everything is okay, merge it after approval

File: Document/Document.ipynb [hunk 1]
```diff
@@ -1228,6 +1229,112 @@
...
+     "output_type": "stream",
+     "text": [
+      "0 {0: 3, 1: 0, 4: 0}\n",
+      "1 {0: 0, 1: 1, 4: 0}\n",
+      "4 {0: 0, 1: 0, 4: 0}\n"
+     ]
+    }
+   ],
+   "source": [
+    "for row,col in cm:\n",
```
Comment (sadrasabouri) [inline Document/Document.ipynb:1262]: `for row, col in cm:`

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
A minor change.

File: Document/Document.ipynb (hunk 1 above)
Comment (sepandhaghighi) [reply Document/Document.ipynb]: Fixed in https://github.com/sepandhaghighi/pycm/pull/428/commits/488970a6ffcaceee93c3ab4f0dc8d6732bccf0ed

**Review by sadrasabouri — APPROVED** [review]

**Review by alirezazolanvari — APPROVED** [review]

**Pull request merged**